libs/pages/application/src/lib/feature/page-settings-deployment-restrictions-feature/page-settings-deployment-restrictions-feature.tsx

```diff
@@ -0,0 +1,175 @@
+              ...serviceParams,
+              deploymentRestrictionId: deploymentRestriction.id,
+              payload,
+            })
+            closeModal()
+          }}
+        />
+      ),
+    })
+  }
+
+  const handleDelete = (deploymentRestriction: ApplicationDeploymentRestriction) => {
+    openModalConfirmation({
+      title: 'Delete Restriction',
+      name: deploymentRestriction.id,
+      isDelete: true,
+      action() {
+        deleteRestriction({
+          serviceId: applicationId,
+          serviceType: 'APPLICATION',
+          deploymentRestrictionId: deploymentRestriction.id,
+        })
+      },
+    })
+  }
+
+  return (
+    <div className="flex flex-col justify-between w-full">
+      <div className="p-8 max-w-content-with-navigation-left">
+        <div className="flex justify-between mb-10">
+          <div>
+            <div className="flex justify-between mb-2 items-center">
+              <h3 className="text-text-700 text-lg">Deployment Restrictions</h3>
+            </div>
+
+            <p className="text-sm text-text-500 max-w-lg">
+              Specify which changes in your repository should trigger or not an auto-deploy of your application
+            </p>
+          </div>
+          <Button dataTestId="add-button" onClick={handleCreate} iconRight={IconAwesomeEnum.CIRCLE_PLUS}>
+            New Restriction
+          </Button>
+        </div>
+        {deploymentRestrictions?.length > 0 ? (
+          <BlockContent title="Deployment restrictions">
+            <div className="flex flex-col gap-3">
+              {deploymentRestrictions.map((deploymentRestriction, i) => {
+                const { id, type, mode, value } = deploymentRestriction
+                return (
+                  <div key={id} className="flex justify-between w-full items-center gap-3">
+                    <InputText
+                      name={`mode_${id}`}
+                      className="shrink-0 grow flex-1"
+                      label="Mode"
+                      value={mode}
+                      disabled
+                    />
+                    <InputText
+                      name={`type_${id}`}
+                      className="shrink-0 grow flex-1"
+                      label="Type"
+                      value={type}
+                      disabled
+                    />
+                    <InputText
+                      name={`value_${id}`}
+                      className="shrink-0 grow flex-1"
+                      label="Value"
+                      value={value}
+                      disabled
+                    />
+                    <ButtonIcon
+                      className="text-neutral-400"
+                      style={ButtonIconStyle.FLAT}
+                      onClick={() => handleEdit(deploymentRestriction)}
+                      dataTestId="edit"
+                      icon={IconAwesomeEnum.WHEEL}
+                    />
+                    <ButtonIcon
```

Comment:
Can you add for this two buttons the same design that Domains section ? Even if we have override for the style, should be more elegant and iso

<img width="718" alt="Capture d’écran 2023-08-24 à 16 02 11" src="https://github.com/Qovery/console/assets/533928/152f358b-8a4d-4918-b1c1-761c484fe880">

Reply:
I copy paste existing design on the Storage settings which was already inconsistent. I'll fix both but style override is clearly an issue we'll have to solve.